packages/website/.npmrc

```diff
@@ -0,0 +1 @@
+strict-peer-dependencies=false
```

Comment:
Is this file necessary in the repo? If we can remove it, let's remove it for now.